apps/web/utils/paths.ts

```diff
@@ -19,4 +19,5 @@ export const paths = {
   privacyPolicy: '/PrivacyPolicy',
   termsAndConditions: '/TermsAndConditions',
   cancellationRights: '/CancellationRights',
+  productWithCoupon: '/study-room-office/office-chair/design-chair-brookhaven-leather-black_105_1003',
```

Comment:
I'm not sure if we really want our tests to depend on the english version of a product from our main system. What happens when the client switches our system with theirs ? Isn't there a way to get a product with a coupon programatically in our tests ?

Reply:
nope, tests are delicate subject,test will only work with test system.
if you want it to be applied to all client system than you need to take into consideration all data needed by all tests